### ISSUE TITLE
SSR doesn't pick up initial language from "lang" query param.

### ISSUE THREAD

**Issue by kakserpom** [mondeja/leptos-fluent#107]
**My config:**
```
[dependencies]
leptos-fluent = "0.0.34"
fluent-templates = "0.9"

[features]
hydrate = [
  "leptos-fluent/hydrate"
]
ssr = [
  "leptos-fluent/ssr",
  "leptos-fluent/actix",
]
```

```
leptos_fluent! {{
        locales: "./locales",
        translations: TRANSLATIONS,
        check_translations: "./src/**/*.rs",
        sync_html_tag_lang: true,
        initial_language_from_url_param: true,
        initial_language_from_navigator: true,
        initial_language_from_accept_language_header: true,
        cookie_name: "lang",
        initial_language_from_cookie: true,
        set_language_to_cookie: true,
    }};
```

**What happens:**
`...?lang=vi` (no cookies) — SSR responds with an English (fallback language) version , and then CSR repaints it in Vietnamese.

**Comment by mondeja** [comment]
Ah, good catch. Currently, leptos-fluent does not takes the language from `lang` url param on the server side.

**Comment by mondeja** [comment]
Fixed on v0.0.35
